test/PgEFCoreCopy.Tests/DbContextExtensionsTest.cs

```diff
@@ -53,6 +54,7 @@ public async Task BasicTest()
         Assert.AreEqual(a[1].Name, "Test2");
         Assert.AreEqual(a[1].CreatedAt.ToString("o"), dateTime.ToString("o"));
         Assert.AreEqual(a[1].IsActive, false);
+        Assert.AreEqual(a[1].ExampleDate.ToString(), date.ToString());

```

Comment:
Is date ToString not locale specific like datetime ToString? "o" is used with datetime to ensure it's ISO formatted..  of course roundtrip test so it should work anyway... but I like ISO.